handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,60 @@
+# Leverage Cloud File Storage with MinIO
+
+SystemLink File Service allows you to configure and use the Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the File Service also works with MinIO, a server-side software storage stack that is compatible with Amazon S3.
+
+To use MinIO with SystemLink, you need SystemLink version 2020 R3 or later.
+
+To use MinIO as a storage provider, set up the MinIO server on a system that you would like to upload files to. This can be the same machine as the SystemLink server or your own dedicated server. Then, configure the File Service to use that server.
+
+## Running MinIO server
+
+1. Download the server application from the [MinIO website](https://min.io/download).
+
+2. From the command line, run the following command to run the server:
+
+    ```bash
+    minio server <PathToRootFolder>
+    ```
+
+    <figure>
+    <img src="../../img/minio-server.png" width="500" />
+    <figcaption>Running a minio server.</figcaption>
+    </figure>
+
+3. When starting the MinIO server for the first time, use a web browser and navigate to the URL that the command line prints out. Create a bucket by clicking the `+` button in the right bottom corner.
+
+    !!! note "Note"
+        - It is recommended to set non-default access key and secret key. You can do that by setting the environment variables `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`. Run the following commands from the command line to start a server pointing at C:\minio with your individual access and secret keys:
```

Comment:
'NI recommends setting a non-default access key and secret key. To do this, define the 'MINIO_ACCESS_KEY' and 'MINIO_SECRET_KEY' variables.'

Since this is recommended, maybe we should make it an actual step? We could mark it as optional if you want.

Reply:
Actually, it is *not* NI who recommends this, but minIO themselves. I'd rather keep it as "It is recommended" but can certainly make it a separate step.

Reply:
Ah! That makes sense. In that case, we might want to say that minIO recommends it, just to avoid liability on our end! We could even link to somewhere in the minIO help that supports this.

Reply:
Moved it upwards in the document, it's step 2 now.